lib/matplotlib/compat/subprocess.py

```diff
@@ -36,6 +36,7 @@
     STDOUT = subprocess.STDOUT
     CalledProcessError = subprocess.CalledProcessError
     check_output = subprocess.check_output
+    TimeoutExpired = subprocess.TimeoutExpired
```

Comment:
This does not work on python2 unless subprocess32 is installed.

```python
Python 2.7.12 (default, Nov  7 2016, 11:55:55) 
[GCC 6.2.1 20160830] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import subprocess
>>> subprocess.TimeoutExpired
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'module' object has no attribute 'TimeoutExpired'
>>> 
```

This looks like it is working on travis because `subprocess32` in an install require (not a run-time require) so it is always getting installed on travis.  I _think_ that there is a way to install mpl such that this will fail.